controllers/admin/AdminModulesController.php

```diff
@@ -1395,10 +1395,32 @@ public function initModal()
     {
         parent::initModal();
 
+        $module = Module::getInstanceByName(Tools::getValue('configure'));
+        $languages = Language::getLanguages(false);
+        $isNewTranslateSystem = $module->isUsingNewTranslationsSystem();
+        $translateLinks = array();
+        foreach ($languages as $lang) {
+            if ($isNewTranslateSystem) {
```

Comment:
I think you could simply inline `$module->isUsingNewTranslationsSystem()` here and get rid of `$isNewTranslateSystem`